infra/fendermint/scripts/promtail.toml

```diff
@@ -0,0 +1,38 @@
+[tasks.create-log-volume]
+script = "docker volume create fendermint-logs"
```

Comment:
How does this work with multiple docker containers, like in the `snapshot-test`? It looks like the same volume will be mounted to all of them, and all of them will try to log to the same filename.

There will be multiple  promtail containers, one for each fendermint container, but they will all try to scrape every file, so there will be multiple containers pushing the same data, no?

Reply:
Aren't the different integration tests each run separately? I think so, at least when looking at our e2e make target:

```
e2e: docker-build | cargo-make
	cd testing/smoke-test    && cargo make --profile $(PROFILE)
	cd testing/snapshot-test && cargo make --profile $(PROFILE)
	cd testing/graph-test    && cargo make --profile $(PROFILE)
```
That should properly create/destroy the docker volume between the tests